kube-runtime/src/controller.rs

```diff
@@ -281,3 +285,33 @@ where
         applier(reconciler, error_policy, context, self.reader, self.selector)
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::{Context, ReconcilerAction};
+    use crate::Controller;
+    use k8s_openapi::api::core::v1::ConfigMap;
+    use kube::Api;
+
+    fn assert_send<T: Send>(x: T) -> T {
+        x
+    }
+
+    fn mock_type<T>() -> T {
+        unimplemented!(
+            "mock_type is not supposed to be called, only used for filling holes in type assertions"
+        )
+    }
```

Comment:
we do have `static_assertions` crate in kube dep tree already so maybe we can do something like
https://docs.rs/static_assertions/1.1.0/static_assertions/macro.assert_impl_any.html

but then again we don't have an easy type here, your test is probably better

Reply:
I don't think that'd work, sadly, since `Controller::run` returns an `impl Stream` (and as such, has an unmentionable type).